### PR TITLE
Feature/description

### PR DESCRIPTION
**Pull request by oyessb**
Added description for jsonstat2 components

### CODE REVIEW

File: PxAPI-2.yml [hunk 1]
```diff
@@ -811,59 +836,71 @@ components:
...
-    #       pattern: "^((19|20)\\d\\d)\\-(0?[1-9]|1[012])\\-(0?[1-9]|[12][0-9]|3[01])$"
     updated:
-        type: string
-        pattern: "^((19|20)\\d\\d)\\-(0?[1-9]|1[012])\\-(0?[1-9]|[12][0-9]|3[01])$"
+      description: See https://json-stat.org/full/#updated
+      type: string
+      pattern: "^((19|20)\\d\\d)\\-(0?[1-9]|1[012])\\-(0?[1-9]|[12][0-9]|3[01])$"
     href:
+      description: Spesification on json-stat.org -> [here](https://json-stat.org/full/#href)
       type: string
```
Comment (likp) [inline PxAPI-2.yml:845]: Looks like a copy past mistake of the misspelling of Specification

**Review by likp — CHANGES_REQUESTED** [review]
Just a misspelling otherwise it looks great!

File: PxAPI-2.yml [hunk 2]
```diff
@@ -811,59 +836,71 @@ components:
...
             decimals:
+              description: The number of decimals in the table cells
               type: integer
             official-statistics:
+              description: Indicates if the data table is included in the official statistics of the organization
               type: boolean
             aggregallowed:
+              description: If the contents of the table cannot be aggregated
               type: boolean
+            refperiod:
```
Comment (PerIngeVaaje) [inline PxAPI-2.yml:879]: Should be on dimension

**Review by PerIngeVaaje — CHANGES_REQUESTED** [review]
Added one comment on misplaced keyword

File: PxAPI-2.yml (hunk 2 above)
Comment (oyessb) [reply PxAPI-2.yml]: Removed refperiod for px. It is defined inside extension for dimenison.

**Review by likp — APPROVED** [review]
Looks great!

**Pull request merged**